src/impl/util.js

```diff
@@ -130,6 +130,14 @@ export function roundTo(number, digits, towardZero = false) {
   return rounder(number * factor) / factor;
 }
 
+export function invertNumber(num) {
```

Comment:
I don't think this is needed. `-num` works as expected and NaN passes through it cleanly. And the results of `parseInteger` are either a number or NaN anyway. I propose just doing this directly in regexParser.js

```js
const maybeNegate = num => hasNegativePrefix ? -num : num;
//...
years: maybeNegate(parseInteger(yearStr))
// etc
```